sound/smooth_swing_v2.h

```diff
@@ -214,13 +260,21 @@ class SmoothSwingV2 : public SaberBasePassThrough {
   };
   Data A;
   Data B;
-
+  Data D;
```

Comment:
We should not use a 'Data' here.
Instead we should use something like:

 RefPtr < BufferedWavPlayer > accent_player_;

instead.

Reply:
Done.